backend/requirements.txt

```diff
@@ -6,3 +6,4 @@ git+https://github.com/patil-suraj/vqgan-jax.git
 git+https://github.com/borisdayma/dalle-mini.git
 tqdm
 flax
+ipywidgets==7.7.0
```

Comment:
Why is this package needed?

Reply:
Without the `ipywidgets ` pkg getting this error. Also getting the same error using local virtualenv

<img width="1446" alt="Screenshot 2022-05-21 at 11 18 40 AM" src="https://user-images.githubusercontent.com/17096858/169637698-d718e246-a754-4814-9d59-caf24b23449a.png">

Reply:
This can get fixed by changing `trange` to `range`

https://github.com/saharmor/dalle-playground/blob/9af9fce6c0e261db30d931450fa1b771d327ebaf/backend/app.py#L102

No need to add additional pkg. 

Let me know your thoughts

Reply:
done